src/libraries/Microsoft.PowerFx.Core/IR/Nodes/BinaryOpKind.cs

```diff
@@ -17,6 +17,11 @@ internal enum BinaryOpKind
         AddDateTimeAndDay,
         AddTimeAndMilliseconds, // Time + Number (Milliseconds)
 
+        SubNumbers,
+        SubDateAndDay,
+        SubDateTimeAndDay,
+        SubTimeAndMilliseconds, // Not yet implemented
```

Comment:
is this used?

Reply:
The first 3 yes. The SubTimeAndMilliseconds op kind has been removed.